http4s-server/src/main/scala/com/avast/server/toolkit/http4s/Http4sRouting.scala

```diff
@@ -0,0 +1,14 @@
+package com.avast.server.toolkit.http4s
+
+import cats.Functor
+import org.http4s.syntax.kleisli._
+import org.http4s.{HttpApp, HttpRoutes}
+
+import scala.language.higherKinds
+
+object Http4sRouting {
```

Comment:
are you planning to add more functionality into `Http4sRouting`? I'm asking, because just calling `.orNotFound` doesn't seem much.

Reply:
Right but the reason why I did it is that you need to know that you need this import `import org.http4s.syntax.kleisli._` which is quite sad and hard to find. Anyway, in the update I will push soon it receives varargs so it gives you a bit more functionality.